.github/workflows/e2e.yaml

```diff
@@ -72,13 +72,31 @@ jobs:
 
         # Verify tool is on our path
         gitsign -h
+    - name: Test Sign and Verify commit
+      run: |
+        set -e
 
         # Sign commit
         git commit --allow-empty -S --message="Signed commit"
 
         # Verify commit
         git verify-commit HEAD
 
+        # Extra debug info
+        git cat-file commit HEAD | sed -n '/BEGIN/, /END/p' | sed 's/^ //g' | sed 's/gpgsig //g' | sed 's/SIGNED MESSAGE/PKCS7/g' | openssl pkcs7 -print -print_certs -text
+    - name: Test Timestamped Commit
+      env:
+        # See https://knowledge.digicert.com/generalinformation/INFO4231.html
+        GITSIGN_TIMESTAMP_AUTHORITY: "http://timestamp.digicert.com"
```

Comment:
https?  humm the docs says http, are we living back in 80's 😂

Reply:
Yeah I was also confused but confirmed that switching this to https does not work. 😅